From d998eecd90d3910055595da7132171d35a62e92a Mon Sep 17 00:00:00 2001
From: Li Zeng <li.zeng@intel.com>
Date: Sat, 27 Oct 2012 10:00:16 +0800
Subject: Add a flag OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled

Bug: 70693
Category: Device Enablement
Domain: AOSP-frameworks
Severity: High
Origin: Internal
Upstream-candidate: Yes

Change-Id: Iafea058d32013db2f68adeec6fadb5002c4809e5
Orig-Change-Id: I38da279d822346dddcd6805df2686992a654ed36
Signed-off-by: Li Zeng <li.zeng@intel.com>
Reviewed-on: http://android.intel.com:8080/83203
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/media/openmax/OMX_IVCommon.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/include/media/openmax/OMX_IVCommon.h b/include/media/openmax/OMX_IVCommon.h
index 96a4396..138fae7 100644
--- a/include/media/openmax/OMX_IVCommon.h
+++ b/include/media/openmax/OMX_IVCommon.h
@@ -162,6 +162,10 @@ typedef enum OMX_COLOR_FORMATTYPE {
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar64x32Tile2m8ka = 0x7FA30C03,
     OMX_SEC_COLOR_FormatNV12Tiled = 0x7FC00002,
     OMX_QCOM_COLOR_FormatYUV420PackedSemiPlanar32m = 0x7FA30C04,
+    
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar =  0x7FA00E00,
+    OMX_INTEL_COLOR_FormatYUV420PackedSemiPlanar_Tiled =  0x7FA00F00,
+ 
     OMX_COLOR_FormatMax = 0x7FFFFFFF
 } OMX_COLOR_FORMATTYPE;
 
-- 
1.7.4.1

